Octokit.Tests/Clients/RepoCollaboratorsClientTests.cs

```diff
@@ -36,6 +36,25 @@ public void RequestsCorrectUrl()
             }
 
             [Fact]
+            public void RequestsCorrectUrlWithApiOptions()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new RepoCollaboratorsClient(connection);
+
+                var options = new ApiOptions
+                {
+                    PageSize = 1,
+                    PageCount = 1,
+                    StartPage = 1
+                };
+
+                client.GetAll("fake", "repo", options);
+
+                connection.Received()
+                    .GetAll<CommitStatus>(Arg.Is<Uri>(u => u.ToString() == "repos/owner/test/collaborators"), Args.ApiOptions);
```

Comment:
I think you made a mistake here. Shouldn't it be `"repos/fake/repo/collaborators"`?